### PR TITLE
Warning about DKIM under specific configuration

### LINKED ISSUE #9782: DKIM signature is omitted when sender and recipient are controlled by the same organization
Hello,

the document should mention that DKIM signature isn't performed on outbound emails when both the sender's domain and the recipient's domain are controlled by the same organization on Office365.

For example, a company owns multiple domains and has mailboxes setup on them. In that case, even if DKIM is enabled and the sender's DNS zone is properly configured, the recipient will see:
```
(...)
Authentication-Results: dkim=none (message not signed) header.d=none;dmarc=none action=none header.from=<sender_domain>;
(...)
```

For context: I spent quite a bit of time trying to make DKIM work, until is stumbled upon this message: https://answers.microsoft.com/en-us/outlook_com/forum/all/office-365-dkim-not-signing/d7b7bd10-7794-4f47-a055-a3a2ba8ad2b0. I couldn't find this information anywhere else.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 249c059f-c3ce-a382-b99e-795a0c1f646a
* Version Independent ID: 5656c114-ae16-6194-cd9f-353b33f4a1e1
* Content: [How to use DKIM for email in your custom domain - Office 365](https://learn.microsoft.com/en-us/microsoft-365/security/office-365-security/use-dkim-to-validate-outbound-email?view=o365-worldwide)
* Content Source: [microsoft-365/security/office-365-security/use-dkim-to-validate-outbound-email.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/security/office-365-security/use-dkim-to-validate-outbound-email.md)
* Service: **microsoft-365-security**
* Sub-service: **mdo**
* GitHub Login: @MSFTTracyP
* Microsoft Alias: **tracyp**

### PR DESCRIPTION
**Pull request by FlorentATo**
Fixes #9782

### CODE REVIEW

File: microsoft-365/security/office-365-security/use-dkim-to-validate-outbound-email.md [hunk 1]
```diff
@@ -286,6 +286,17 @@ Wait a few minutes before you follow these steps to confirm that you have proper
 
 - Look for the Authentication-Results header. While each receiving service uses a slightly different format to stamp the incoming mail, the result should include something like **DKIM=pass** or **DKIM=OK**.
 
+> [!IMPORTANT]
+> The DKIM signature is **omitted** under any of these conditions:
+> - both the sender and the recipient email address use the same domain;
+> - both the sender and the recipient email address use different domains that are controlled by the same organization.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/use-dkim-to-validate-outbound-email.md]:
```suggestion
> - Both the sender and the recipient email address use the same domain.
> - Both the sender and the recipient email address use different domains that are controlled by the same organization.
```

File: microsoft-365/security/office-365-security/use-dkim-to-validate-outbound-email.md [hunk 2]
```diff
@@ -286,6 +286,17 @@ Wait a few minutes before you follow these steps to confirm that you have proper
...
+>
+> In both cases, the header will look like something like this:
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/use-dkim-to-validate-outbound-email.md]:
```suggestion
> In both cases, the header will look similar to this:
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @FlorentATo, kindly commit the suggested changes. Many thanks.

**Comment by FlorentATo** [comment]
@JohanFreelancer9 for final review, please.

**Comment by yogkumgit** [comment]
@MSFTTracyP Please merge this PR. Thanks

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@MSFTTracyP Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@tiburd Please help us.

**Comment by yogkumgit** [comment]
@MSFTTracyP Please merge this PR. Thanks

**Comment by FlorentATo** [comment]
@MSFTTracyP, @tiburd any chance to get to PR merged anytime soon ? Thank you.

**Comment by tiburd** [comment]
@MSFTTracyP 
Hi,
Please review and merge if you approve the changes.
Thanks!

**Comment by FlorentATo** [comment]
Any update on this @MSFTTracyP @tiburd @yogkumgit @JohanFreelancer9 ?

**Comment by FlorentATo** [comment]
Replaced by https://github.com/MicrosoftDocs/microsoft-365-docs/pull/10472

**Pull request closed without merging**